opendebates/views.py

```diff
@@ -255,21 +255,11 @@ def vote(request, id):
     return redirect(url)
 
 
-@rendered_with("opendebates/list_ideas.html")
 @allow_http("GET", "POST")
```

Comment:
The `@rendered_with` is needed if returning a dictionary at line 270 needs to work.